src/Console/Command/CheckCommand.php

```diff
@@ -18,15 +21,124 @@
 )]
 final class CheckCommand extends Command
 {
-    public function __construct()
-    {
+    public function __construct(
+        private readonly RuntimeCheck $runtimeCheck
+    ) {
         parent::__construct();
     }
 
+    protected function configure(): void
+    {
+        $this
+            ->setHelp('Command to performs a check of the runtime environment')
+            ->addOption(
+                'skip',
+                null,
+                InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
+                'Skip check for different areas. E.g. for "fpm" .',
```

Comment:
maybe we add a more detailed description on what exactly can be "skipped". afaik these can be scopes like 'fpm' or 'cli' or direct checkers like 'logger' or 'php'.